content/api/commands/session.md

```diff
@@ -396,13 +404,6 @@ function validate() {
   cy.visit('/account', { failOnStatusCode: false })
   cy.url().should('match', /^/account/)
 }
-
-// Or just return false if the session is invalid
```

Comment:
Do we not want an example for "the `validate` function returns a Promise that resolves to `false` or rejects" anymore?

Reply:
I'm going to do more comprehensive evaluation for the docs for the v12 release.